CITATION.cff

```diff
@@ -3,6 +3,6 @@ message: "If you use this software, please cite it as below."
 authors:
 - name: "The Agda Community"
 title: "Agda Standard Library"
-version: 2.1
-date-released: 2024-07-17
+version: 2.1.1
+date-released: 2024-09-03
```

Comment:
I'll update it to today for now, but as above, I'll leave this to @MatthewDaggitt to fix up for the 'actual' release, in the interests of merging sooner rather than later.